fractals/sunqua-peak/index.md

```diff
@@ -151,14 +155,18 @@ The boss has multiple phases themed around the different elements air, fire, wat
 #### Air Phase <Label>100%-66%</Label>
 <Grid>
 <GridItem xs="12" sm="7">
-After crossing the green line, the fight starts. After about one second, Ai dashes to a randomly chosen side and casts a set of line attacks. During this time, a CC bar appears, which should only attemped to be broken in organized groups (due to the very small window of opportunity that requires coordination). After the line attacks, she will dash to the middle and target each player with an orange AoE, forcing the group to split up to minimize damage taken. When the circles disappear, group up (avoiding any subsequent lines or explosions) and start bursting. If you have not broken the 1st CC bar, a second will appear, as Ai starts casting a series of cone attacks. Breaking the bar at the last second is key for maximum DPS. To do this, you need to remember the pattern of upcoming attacks when the CC bar is active. That pattern is 2 cones, a multihit circles (that must be dodged), another cone and finally, a red AoE that deals massive damage and forces the group to split. Break the bar when the red circle appears and stay in the middle, upkeeping DPS.
-
-If you do not manage to bring her to 66%, she will dash twice and get <Effect name="Invulnerability"/>, summoning a tornado, like the Voice of the Mountain. Approach this phase as you did before, only this time, you must disperse 8 clouds quickly, or Ai will cast a massive AoE that wipes the group. Keep in mind that a set of 3 <Uncategorized name="chargedleap"/> orbs spawn in each cardinal direction. Split up accordingly and move in the same direction to interrupt this attack as quickly as possible.
-
+After crossing the green line, the fight starts. After about one second, Ai dashes to a randomly chosen side and casts a set of line attacks. During this time, a CC bar appears, which should only attemped to be broken in organized groups (due to the very small window of opportunity that requires coordination). After the line attacks, she will dash to the middle and target each player with an orange AoE, forcing the group to split up to minimize damage taken. When the circles disappear, group up (avoiding any subsequent lines or explosions) and start bursting. If you have not broken the 1st CC bar, a second will appear, as Ai starts casting a series of cone attacks. 
```

Comment:
No no no! :(
Do not try to break first bar, breaking it will result in no 2ed bar.
And not having 2ed bar meaning you cannot break it to avoid red circle